lib/parse-font.js

```diff
@@ -46,6 +49,12 @@ module.exports = str => {
   const sizeFamily = sizeFamilyRe.exec(str)
   if (!sizeFamily) return // invalid
 
+  const names = sizeFamily[3]
+    .match(familyRe)
+    // remove actual bounding quotes, if any, unescape any remaining quotes inside
+    .map(s => s.trim().replace(unquoteRe, '$1').replace(unescapeRe, '$1'))
+    .filter(s => !!s)
+
```

Comment:
Isn't that the same as this? We already know it matches `familyRe`:

```suggestion
   // remove actual bounding quotes, if any, unescape any remaining quotes inside
  const names = sizeFamily[3].trim().replace(unquoteRe, '$1').replace(unescapeRe, '$1'))
```

Reply:
Oh my bad, now I see it. It's for comma-separated fonts.